alertwildfire_live_recording.py

```diff
@@ -125,7 +137,7 @@ def fetchAllCameras(googleServices, camera_names_to_watch):
         except Exception as e:
             logging.error('Failed to delete temporaryDir %s. %s', temporaryDir.name, str(e))
             pass
-
+        print('toc = '+str( time.time()-tic))
```

Comment:
I like the timing information, but please use logging.warning() instead of print and add some descriptive text.